axe.d.ts

```diff
@@ -146,6 +153,14 @@ declare namespace axe {
 		}[];
 		cleanup?(callback: Function): void;
 	}
+	interface VirtualNode {
```

Comment:
This isn't exported on `axe-core`. Shouldn't this be `class AbstractVirtualNode`?

Reply:
The ts file seems to be taking things from the `axe.d.ts` file and not the `axe.js` file. I could import the `axe.VirtualNodeProps` without problems.